.github/repo_policies/bot_approved_files.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think a plain text file with a path per line would be simpler here and would look more similar to CODE OWNERS files which are related to this. But that something we could consider changing in a follow up.

Reply:
Ah okay, yeah happy to discuss this more offline. Just thought json was a more standard data format and easier to parse.

Reply:
The advantage of plain text over JSON is that you can also add comments which could be useful to document things. Even with that [the JSON parsing](https://github.com/dfinity/public-workflows/blob/3e765100cf424d735ce09fb37d6b95550f8aa0e0/reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py#L49-L56) you have can be simplified to just: 
```python
approved_files = [line for line in config_file.splitlines() if not line.strip().startswith("#")]
```

Reply:
Another advantage of a file per line is that we could potentially extend this to specify which bot is allowed to modify which file like:
```
ic-os/boundary-guestos/context/docker-base.prod @sa-github-api
ic-os/guestos/context/docker-base.dev           @sa-github-api
ic-os/guestos/context/docker-base.prod          @sa-github-api
ic-os/hostos/context/docker-base.dev            @sa-github-api
ic-os/hostos/context/docker-base.prod           @sa-github-api
ic-os/setupos/context/docker-base.dev           @sa-github-api
ic-os/setupos/context/docker-base.prod          @sa-github-api
testnet/mainnet_revisions.json                  @sa-github-api
```
but you could imagine having [different bots](https://github.com/dfinity/public-workflows/blob/main/reusable_workflows/check_membership/check_membership.py#L7-L12) in there as well.